osf/models/registrations.py

```diff
@@ -82,11 +83,22 @@ def find_failed_registrations():
         return stuck_regs
 
     @property
```

Comment:
hm, I did intend for it to be a property, I'm not passing anything in, each DraftRegistration and Registration is practically associated with one schema.  DraftRegistration already has a `registration_schema` relationship, but the corresponding field on Registrations is an m2m field, so I wanted the property to pull off the first one (we really should make that relationship a foreign key).

I think I changed the name to avoid conflicting with the name on the DraftRegistration property, but still having one field that could be defined on both models. I also wanted to be able to throw a NotImplementedError if it wasn't defined wherever the mixin was used.

Does keeping it a property but changing the name to something that doesn't have `get_` in it, work?

Reply:
Oh whoops, I was thinking this was a serializer, where DRF will default to using a `get_<field>` method for `SerializerMethodField`. But yeah, I'd expect a name with an action verb to be a method, so if you don't mind either making it a method or renaming to something more noun-y, that'd be nice.

(personally, I think properties should be super lightweight (e.g. no DB queries), since they're so easy to use over and over without thinking, but that's not at all the convention in this codebase, so i'm fine with either)